src/Perpetuum/Services/MissionEngine/Missions/MissionInProgress.cs

```diff
@@ -1461,10 +1461,10 @@ public double GetDifficultyReward
 
         public double GetParticipantBonusModifier()
         {
-            return computeParticipantBonusMultiplier(this._participants.Count);
+            return ComputeParticipantBonusMultiplier(GetParticipants().Count);
```

Comment:
This is the actual fix, rest is refactoring and properly accessing the _participants collection via its getter

Reply:
Fix is good. This entire class needs refactoring and restructuring though; a job for ol' Ron I think.